en/docs/chapter_searching/searching_algorithm_revisited.md

```diff
@@ -1,48 +1,48 @@
 # Search algorithms revisited
 
-<u>Searching algorithms (searching algorithm)</u> are used to search for one or several elements that meet specific criteria in data structures such as arrays, linked lists, trees, or graphs.
+<u>Searching algorithms (search algorithms)</u> are used to retrieve one or more elements that meet specific criteria within data structures such as arrays, linked lists, trees, or graphs.
 
-Searching algorithms can be divided into the following two categories based on their implementation approaches.
+Searching algorithms can be divided into the following two categories based on their approach.
 
 - **Locating the target element by traversing the data structure**, such as traversals of arrays, linked lists, trees, and graphs, etc.
-- **Using the organizational structure of the data or the prior information contained in the data to achieve efficient element search**, such as binary search, hash search, and binary search tree search, etc.
+- **Using the organizational structure of the data or existing data to achieve efficient element searches**, such as binary search, hash search, binary search tree search, etc.
 
```

Comment:
- 'prior information contained in the data' can be shortened. We could also say 'existing information' instead of 'existing data'.
- 'searches' or we could say 'an efficient element search'

Reply:
If so, I think the translation in line 7 should also adjust the sentence structure to be more consistent.